modules/@apostrophecms/area/index.js

```diff
@@ -51,7 +51,14 @@ module.exports = {
             self.warnMissingWidgetType(type);
             throw self.apos.error('invalid');
           }
-          widget = await sanitize(widget);
+          try {
+            widget = await sanitize(widget);
+          } catch (e) {
+            if (livePreview) {
+              return 'aposLivePreviewSchemaNotYetValid';
```

Comment:
The result is HTML, not JSON, so we return a special string to avoid a status code that shows up as an "error" in the browser console. When previewing an error from sanitize is sometimes normal, so we return this code to indicate it's OK to do nothing.